stack.py

```diff
@@ -0,0 +1,63 @@
+from static_array import StaticArray
+
+
+class Stack:
+    def __init__(self, size):
+        # TC: O(1)
+        self.size = size
+        self.array = StaticArray(size)
```

Comment:
The TC of init is `O(size)` as you're initialising StaticArray which creates a list internally. As you're using it, you inherit its time complexity too!